runtime/near-vm-logic/src/alt_bn128.rs

```diff
@@ -47,20 +28,31 @@ pub fn alt_bn128_g1_multiexp_sublinear_complexity_estimate(n_bytes: u64, discoun
 
```

Comment:
These constants seem to be estimated. They should be together with the other estimated costs, not here.